src/zrc/api/schema.py

```diff
@@ -5,7 +5,7 @@
 
 from .kanalen import KANAAL_ZAKEN
 
-description = f"""Een API om een zaakregistratiecomponent (ZRC) te benaderen.
+description = f"""Een API om een Zaken API (ZRC) te benaderen.
```

Comment:
@joeribekker is this replacement correct?

Reply:
nee deze niet, goed opgemerkt :)

Reply:
heb hem teruggedraaid